AnnoyancesFilter/Popups/sections/antiadblock.txt

```diff
@@ -631,6 +631,7 @@ reddit.com#@#.ADBAR
 ! SECTION: Anti-adblock - Regular rules
 ! Do not forget about '!#safari_cb_affinity' section for exclusions
 !
+||www3.doubleclick.net/|$redirect=nooptext,domain=photosonic.writesonic.com
```

Comment:
Better with content type
```suggestion
||www3.doubleclick.net/|$xmlhttprequest,redirect=nooptext,domain=photosonic.writesonic.com
```
also can be used this scriptlet(also works in Safari)
`photosonic.writesonic.com#%#//scriptlet('prevent-fetch', 'www3.doubleclick.net')`

Reply:
Done.